subsys/dfu/src/dfu_target_mcuboot.c

```diff
@@ -65,6 +65,38 @@ int dfu_ctx_mcuboot_set_b1_file(char *file, bool s0_active, char **update)
 	return 0;
 }
 
+#define MODULE "dfu"
+#define FILE_FLASH_IMG "mcuboot/flash_img"
+static int store_offset(void)
```

Comment:
Can this be a more specific name? `store_flash_img`, `backup_flash_img`, `backup_progress` or something like that? Also, function docs :).

Reply:
Woops forgot to update the name of this

Reply:
Renamed and added docs